### ISSUE TITLE
removing digits

### ISSUE THREAD

**Issue by mleblanc321** [WheatonCS/Lexos#148]
jesse is working on "removing digits" ... i think the solution is like punctuation;  that is, we make a digitMAP of all unicode characters that are digits in the unicode character set:

unicodedata.category(unichr(i)).startswith('N')

and then nuke those in the digitMAP  (of couse, caching the digitMAP so we only have to do it once and not every time, like we do with punctuation)

** make sense?

**Comment by jaronson13** [comment]
We were able to create a function called remove_digits that successfully removes all unicode characters that are digits from a block of text. We came across a small issue:

The call chardet.detect(request.data[:500]) does not seem to pick up on the unicode encoding of a small text file. For example, I wrote the word "test" with three Eastern Arabic digits between the English letters and tried scrubbing this file. The chardet interpreted the digits as TIS-620 encoding which corresponds to Thai characters. So without a larger sample of text chardet() has trouble detecting some encoding. I'm going to look into character detection a little more.

We also discussed a small GUI addition to the scrubber page that would enable the user to only remove Arabic numerals 0-9 if they wanted, rather than all unicode digits. This could be a possible future addition
